corehq/motech/openmrs/tests/test_atom_feed.py

```diff
@@ -89,6 +104,66 @@ def test_success(self):
         self.assertEqual(patient_uuid, 'e8aa08f6-86cd-42f9-8924-1b3ea021aeb4')
 
 
+class ImportEncounterTest(SimpleTestCase, TestFileMixin):
+    file_path = ('data',)
+    root = os.path.dirname(__file__)
+
+    def setUp(self):
+        self.case = CaseMock(
+            name='Randall',
+            type='patient',
+            _id='abcdef',
+        )
+        self.repeater = Mock()
+        self.repeater.domain = 'test_domain'
+        self.repeater.get_id = '123456'
+        self.repeater.white_listed_case_types = ['patient']
+        self.repeater.openmrs_config.form_configs = [OpenmrsFormConfig.wrap({
+            "doc_type": "OpenmrsFormConfig",
+            "xmlns": "http://openrosa.org/formdesigner/9481169B-0381-4B27-BA37-A46AB7B4692D",
+            "openmrs_visit_type": "c22a5000-3f10-11e4-adec-0800271c1b75",
+            "openmrs_encounter_type": "81852aee-3f10-11e4-adec-0800271c1b75",
+            "openmrs_observations": [{
+                "doc_type": "ObservationMapping",
+                "concept": "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "value": {
+                    "doc_type": "FormQuestion",
+                    "form_question": "/data/height"
+                },
+                "case_property": "height"
+            }]
+        })]
+        response = Mock()
+        response.json.return_value = self.get_json('encounter')
+        self.repeater.requests.get.return_value = response
+
+    def test_import_encounter(self):
+        """
+        Importing the given encounter should update the case's "height" property
+        """
+        with patch('corehq.motech.openmrs.atom_feed.submit_case_blocks') as submit_case_blocks_patch, \
+                patch('corehq.motech.openmrs.atom_feed.importer_util') as importer_util_patch:
+            importer_util_patch.lookup_case.return_value = (self.case, None)
+
+            import_encounter(self.repeater, 'c719b87f-d221-493b-bec7-c212aa813f5d')
+
+            case_block_re = """
+<case case_id="abcdef" »
+      date_modified="[\\d\\-T\\:\\.Z]+" »
+      xmlns="http://commcarehq.org/case/transaction/v2">
+  <update>
+    <height>105</height>
+  </update>
+</case>
+"""
```

Comment:
weird indent, you can use `inspect.cleandoc` if you need to get rid of the indent